### PR TITLE
[Dev] Apply upstream patches

### PR DESCRIPTION
**Pull request by carlopi**

### CODE REVIEW

File: .github/workflows/Linux.yml
Comment (samansmink) [inline .github/workflows/Linux.yml:1]: This one should stay: it's used to test against a postgres server

**Review by samansmink — COMMENTED** [review]
1 small comment, the rest looks good!

File: .github/workflows/Linux.yml
Comment (carlopi) [reply .github/workflows/Linux.yml:1]: Thanks! Removing Linux.yml was a bit too far, thanks for having another look.

**Comment by carlopi** [comment]
I think this should be ready to go in.
Failure on the Linux test is due to the fact that this was not properly tested yet after changes to main forbidding the `"MyColumn:1"` syntax (should be probably changed to `"tbl_with_case_sensitive_columns.MyColumn"`)

**Comment by Mytherin** [comment]
Yolo

**Pull request merged**